test/integration/stats_integration_test.cc

```diff
@@ -310,7 +311,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
     // https://github.com/envoyproxy/envoy/issues/12209
     // EXPECT_MEMORY_EQ(m_per_cluster, 44949);
   }
-  EXPECT_MEMORY_LE(m_per_cluster, 47500); // Round up to allow platform variations.
+  EXPECT_MEMORY_LE(m_per_cluster, 47700); // Round up to allow platform variations.
```

Comment:
this whole test is now gone, once you merge, so you can just keep the edit for the real symbol-tables test below, and drop this one.